config/v1/types_image.go

```diff
@@ -162,6 +162,8 @@ type RegistryLocation struct {
 }
 
 // RegistrySources holds cluster-wide information about how to handle the registries config.
+//
+// +kubebuilder:validation:XValidation:rule="self.blockedRegistries.size() > 0 ? self.allowedRegistries.size() == 0 : true",message="Only one of BlockedRegistries or AllowedRegistries may be set"
```

Comment:
Would this be an easier validation to understand? 
```suggestion
// +kubebuilder:validation:XValidation:rule="self.blockedRegistries.size() == 0 || self.allowedRegistries.size() == 0",message="Only one of BlockedRegistries or AllowedRegistries may be set"
```

Reply:
updated